serve-api/handlers/order.go

```diff
@@ -0,0 +1,68 @@
+package handlers
+
+import (
+	"bytes"
+	"encoding/csv"
+	"fmt"
+	"log"
+	"net/http"
+	"strconv"
+	"time"
+)
+
+func (h *BaseHandler) GetOrder(w http.ResponseWriter, r *http.Request) {
```

Comment:
Although not part of the ticket, I created this get endpoint as a way of manually testing everything worked, as hitting the report endpoint which downloads a csv was becoming annoying. I figured there wasn't any harm in keeping it since we're bound to need a 'find by id' type func at some point :)